src/Makefile.test.include

```diff
@@ -36,7 +36,7 @@ JSON_TEST_FILES = \
   test/data/g1_compressed.json \
   test/data/g2_compressed.json
 
-#RAW_TEST_FILES = test/data/alertTests.raw
```

Comment:
It seems to me variables `RAW_TEST_FILES` and `GENERATED_TEST_FILES` can be dropped, correct?